content/analyst/inbox-and-design-tracker/inbox-tracker-reference-guide.md

```diff
@@ -0,0 +1,95 @@
+
+### **Basic Platform Knowledge**
+
+* **[What is a user panel?](/analyst/general/what-is-a-user-panel)** Learn more about our panel and how it makes our data unique.
+* **[Common Terminology & Definitions.](/analyst/general/common-terminology-and-definitions)** This article covers our most common terms, phrases and words and what they actually mean within the platform.
+* **[Naming conventions:](/analyst/general/name-mapping-sparkpost-vs-validty)** Validity/Return Path to Inbox Tracker
+
+---
+
+### Configuring your Account
+
+* **Set up your profiles.** Profiles are a way for you to organize your sending domains and limit the deliverability data you want to see. Click in the text box in the top left corner to view and switch between profiles. You can edit or create additional profiles by clicking the pencil icon to the right of this text box.
+
+![](media/inbox_tracker_reference_guide/image_0.png)
+
+* **Outlook SNDS Integration.** If you send on dedicated IPs, you can bring in SNDS data to understand how Outlook is rating you as a sender. Here is a [set of guides](/analyst/inbox-and-design-tracker/microsoft-snds-set-up-and-configuration) on how to to set up your Outlook SNDS account and how to bring that data into Inbox Tracker.
+
+![](media/inbox_tracker_reference_guide/image_1.png)
+
+*SNDS renders a spam verdict on email from your IP(s):* 
+
+*Green: up to 10% of the time* 
+
+*Yellow: between 10% and 90% of the time* 
+
+*Red: more than 90% of the time* 
+
+* **Google Postmaster Tools Integration.** Similarly to Outlook SNDS, Google Postmaster allows you to see how Google is rating you as a sender. If you already have a Google Postmaster account, here is a [guide](/analyst/inbox-and-design-tracker/how-to-set-up-google-postmaster-tools) on integrating its data into the platform. If you do not have one configured, refer to [this guide](https://support.google.com/mail/answer/6227174?hl=en) from Google to get started.
+* **Begin seeding.** In addition to our panel, you can send to a seed list to get better deliverability coverage across your campaigns. Refer to our [seeding guide(s)](/analyst/inbox-and-design-tracker/inbox-design-tracker#seeding) for guidance on integrating these addresses into your mail streams to increase the amount of campaign-level data.
+* **Setting up alerts.** If you want to monitor your deliverability, traps, blocklists, etc. from afar, alerts are a great way to do that. Here is [a guide](/analyst/inbox-and-design-tracker/getting-started-with-alerts) on how to create a base-set of alerts to begin monitoring changes in your set up or performance.
+
+![](media/inbox_tracker_reference_guide/image_2.png)
+
+*Alerts can be configured to go through your email or via [Slack](/analyst/inbox-and-design-tracker/how-to-configure-slack-for-inbox-tracker)*
```

Comment:
`/analyst/inbox-and-design-tracker/how-to-configure-slack-for-inbox-tracker` -> `/analyst/inbox-and-design-tracker/configuring-slack-for-inbox-tracker/`